### PR TITLE
make init scripts consistent and stop seahub before seafile, including l...

### PR DESCRIPTION
**Pull request by caniwi**
...ogging date and time to init logs

### CODE REVIEW

File: deploy/start_seafile_at_system_bootup.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ and **seafile\_dir** accordingly)
     user=haiwen
 
     # Change the value of "seafile_dir" to your path of seafile installation
-    seafile_dir=/data/haiwen
+    # usually the home directory of $user
+    seafile_dir=`grep ^${user}: /etc/passwd | awk  -F: ' { print $6 } '`
```
Comment (lins05) [inline deploy/start_seafile_at_system_bootup.md]: I don't think it's good to write it like this. A lot of users actually installs seafile to places like `/opt/seafile`, `/var/lib/seafile`, besides, we use `/data/hawen` at almost everywhere else of the documeentation. So we'd better keep it as it was.

File: deploy/start_seafile_at_system_bootup.md [hunk 2]
```diff
@@ -223,7 +235,8 @@ is executed by the system at bootup, so we start seafile/seahub there.
     user=haiwen
 
     # Change the value of "script_path" to your path of seafile installation
-    seafile_dir=/data/haiwen
+    # usually the home directory of $user
+    seafile_dir=`grep ^${user}: /etc/passwd | awk  -F: ' { print $6 } '`
```
Comment (lins05) [inline deploy/start_seafile_at_system_bootup.md]: Same as above.

**Comment by lins05** [comment]
The other changes looks good to me.

**Comment by caniwi** [comment]
Made the changes requested.

**Pull request merged**